docs/UserGuide.md

```diff
@@ -147,6 +147,42 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Adding new Project : `addP`
```

Comment:
`Adding new Project` should be `Adding a new project`. Note the capitalization of `project`.